src/ConnectedMode/UI/ServerSelection/ServerSelectionWnd.xaml

```diff
@@ -0,0 +1,126 @@
+﻿<Window x:Class="SonarLint.VisualStudio.ConnectedMode.UI.ServerSelection.ServerSelectionWnd"
+                         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                         xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+                         xmlns:ui="clr-namespace:SonarLint.VisualStudio.ConnectedMode.UI.Resources"
+                         Title="{x:Static ui:UiResources.ServerSelectionWndTitle}" 
+                         HorizontalAlignment="Center"
+                         SizeToContent="WidthAndHeight"
+                         ResizeMode="NoResize"
+                         MaxWidth="750"
+                         MaxHeight="500"
+                         Icon="pack://application:,,,/SonarLint.VisualStudio.ConnectedMode;component/UI/Resources/sonarlint_32px.ico"
+                         Background="{DynamicResource {x:Static vsShell:VsBrushes.ComboBoxBackgroundKey}}"
+                         FontSize="{DynamicResource {x:Static vsShell:VsFonts.Environment122PercentFontSizeKey}}"    
+                         FontFamily="{DynamicResource {x:Static vsShell:VsFonts.EnvironmentFontFamilyKey}}"
+                         x:Name="This">
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="pack://application:,,,/SonarLint.VisualStudio.ConnectedMode;component/UI/Resources/Styles.xaml"/>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid DataContext="{Binding ElementName=This, Path=ViewModel}" Margin="10">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+
+        <GroupBox Grid.Row="0" Header="{x:Static ui:UiResources.ChooseConnectionTypeLbl}">
```

Comment:
Header text is not adjusting to a dark theme, it doesn't change to white text color.

Reply:
I moved the comment as part of this [UI task](https://sonarsource.atlassian.net/browse/SLVS-1340)